examples/extension/build.gradle

```diff
@@ -11,6 +11,8 @@ plugins {
   See https://imperceptiblethoughts.com/shadow/ for more details about Shadow plugin.
    */
   id "com.github.johnrengelman.shadow" version "6.1.0"
+
+  id "io.opentelemetry.instrumentation.javaagent-codegen" version "0.1.0"
```

Comment:
Is 0.1.0 some temporary released version? Are we going to version gradle plugins the same way we do all other artficats?

Reply:
Currently this module is totally independent from the main build. I don't have a strong opinion on its versioning atm